### DISCUSSION TITLE
Saucedacity 1.2 "Tenacious S"

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#41] [Announcements]
Long in the making, Saucedacity 1.2 has finally been released! :tada:

We have a couple of things to go over here in this release announcement, but for now, I bring to you our old friend, the change log:

# Change Log

## New
* Added a new default theme. Hopefully you guys like it!

## Changed
* We now use Tenacity's libmad fork instead of Audacity's.
  * This was a part of an effort to move away from Conan, but we have an alternative solution that will be implemented soon after this release instead.
* **On Linux and similar, Saucedacity's default preference directory has changed to $XDG_CONFIG_HOME/saucedacity (or `~/.config/saucedacity`)**. If you are migrating from Saucedacity 1.1, it is recommended you copy your `.Saucedacity-data` to this new directory in order to avoid accidentally "resetting" your preferences. **Windows users don't have to worry about this**.

## Fixed
* FFmpeg now works again ([#22](https://github.com/saucedacity/saucedacity/issues/22)).
* The track view is no longer cut off at the bottom of the window ([#20](https://github.com/saucedacity/saucedacity/issues/20))

## Removed
* Removed a useless `IsSaucedacity` preference in `saucedacity.cfg`

# Saucedacity Going Forward
Not too long ago, I began talking with the people over at Tenacity. These talks have progressed well, and I've even received advice from one of them (shoutout to @n0toose for your advice!). But what have we been talking about? Well, I've made a major proposal, and we are going to start merging parts of Tenacity into Saucedacity. From then on (well, not exactly; I'll detail more below), we'll be continuing under the name "Tenacity". We'll practically look like Tenacity (as we'll technically be part of Tenacity then) but we'll still have the code base and the spirit of Saucedacity under the hood.

## The Start of my Involvement in Tenacity
*Note: I don't know if "Involvement" is the right word, but this is the start of my talks in the Tenacity community anyways.

It all started in [#21](https://github.com/saucedacity/saucedacity/issues/21), where @FrostKnight discovered this project and introduced us to Tenacity. Just a little while after, I decided to get involved and try to welcome anyone to the project. Then, a more serious proposal came...

## The Major Proposal
Then, I finally [proposed this major proposal](https://github.com/tenacityteam/tenacity/discussions/733), in which we could either merge Tenacity into Saucedacity or the two into a new fork. Since making the proposal, I've made several decisions regarding Saucedacity, and some attention has been brought to us by Tenacity. Because of that, I've seen some people chip in about our name. The consensus: the name seemed like a joke. Therefore, I knew that if I wanted to continue on with this project, I'd have to change the name. I have finally chosen the path of reusing Tenacity's name, so we'll be called that from now on.

## The Plan Going Forward
While the above was a brief uncovering of events, it should nevertheless provide an accurate summary. Now, what are we going to do going forward? Well, that's what I'll detail right now.

The first thing is that I will take a break. It will be a small one (maybe for a day or two). I kind of want to rest my brain for coding and try to avoid being on a computer that much. Either way, I have some stuff coming up, so this break comes at a perfect time. Of course, I'll still be participating in discussions and messages.

Once my break is over, I will be immediately working towards the next release. This will be the first release under the Tenacity name. and I hope that we make some good progress from then on.

Finally, in case you haven't realized already, **this will be technically the last Saucedacity release that there will ever be**. For those who have stuck around since the early days, it might be hard to accept this change. But don't worry! We'll still be active, just under a different name, and with some added features! :)

This quote explains it well:
> “Don't cry because it's over, be happy because it happened”

― Dr. Suess

I've heard of this quote once before and wanted to use this because this explains my thoughts quite well. However, there's one thing: Saucedacity isn't over. We're just continuing development under a new name.

Now, there are some things to be resolved. We are still discussing which organization to use and similar details. There's also the deal of existing funds from donations. I don't plan to accept for now, but maybe some time in the future.

# Final Words
Before I wrap this long announcement up, I'd like to say a couple of words before we start our new journey. For those who have stopped by, I thank you just for stopping by and checking it out. You were the ultimate motivation for me to work on Saucedacity because you were interested. I never expect anyone to be interested in any work I do, yet you and others saw what I did and were interested. For that, I give you my ultimate thanks and appreciation.

To those of Tenacity, I look forward to working with you.You guys have done some amazing work on Tenacity this far, and I want to continue your work. As one example, I've looked at [this POC PR](https://github.com/tenacityteam/tenacity/pull/608) that looks to automatically set the theme for Windows. I have commented on this and see this PR's potential.

Finally, I have a list of things that I current have on my mind to merge into Saucedacity's code base. **This is definitely not a definitive list of features I will merge into Saucedacity's code base**. We will definitely spend time merging features over from Tenacity's original code base over to Saucedacity's code base. This is the current list:

* Translations (as suggested by [#10](https://github.com/saucedacity/saucedacity/issues/10))
* **Build system** (this was the prerequisite before starting Qt development; since Tenacity's build system removes Conan, this will mean the start of Tenacity 2.0, written using Qt).
* Themes (I plan to still keep Saucedacity's theme as an extra theme for now)
* Dynamic theming support, e.g., where Tenacity adapts to the system's theme (currently available [as a PR for Windows only](https://github.com/tenacityteam/tenacity/pull/608), but we are definitely in discussion about it).

If you have suggestions as to what other changes in Tenacity there are that should be merged into Saucedacity's code base, feel free to open an issue and we'll go over it.

---

So that's it. The last release under the name Saucedacity. It's time to start working on the future of Tenacity and the first ever release of Tenacity, which is long overdue.

<hr /><em>This discussion was created from the release <a href='https://github.com/saucedacity/saucedacity/releases/tag/v1.2'>Saucedacity 1.2 "Tenacious S"</a>.</em>

**Comment by poyribn** [comment]
¿will there be a "generic Linux" version?
instead of _.AppImage_, these end in _.x86_ for 32-bit computers and in _.x86_64_ for 64-bit computers

**Comment by generic-pers0n** [comment]
I can release a "genetic Linux" version like I have done in the past. I can do it for x86\_64 for this release.

In the future, we can add 32-bit generic Linux builds alongside AppImages.

**Reply by poyribn** [reply]
thank you for adding the generic Linux version.
i will try it when i need an audio editor, which is soon but not immediately.

**Reply by generic-pers0n** [reply]
> thank you for adding the generic Linux version. i will try it when i need an audio editor, which is soon but not immediately.

No problem! Hopefully you like it, although I do believe that there are some things I can improve on.
